.github/workflows/end2end.yaml

```diff
@@ -81,6 +81,9 @@ env:
   RING_S3C_ENDPOINT: http://s3c.local:8000
   RING_S3C_BACKEND_SOURCE_LOCATION: rings3cbackendingestion
   RING_S3C_INGESTION_SRC_BUCKET_NAME: ingestion-test-src-bucket-${{ github.run_id }}
+  RING_S3C_BACKEND_SOURCE_NON_VERSIONED_LOCATION: rings3cbackendingestionnonversioned
+  RING_S3C_INGESTION_SRC_NON_VERSIONED_BUCKET_NAME: ingestion-test-src-non-versioned-bucket-${{ github.run_id }}
```

Comment:
the github run_id is an incrementing number (currently 11 char), meaning the bucket name length here is already 52 characters. To be future proof, maybe good to ensure the length cannot exceed 63 chars (the max for an s3 bucket name)